### PR TITLE
TELEGRAF-3: Adding README to ah_wireless

### PR DESCRIPTION
**Pull request by chayan-04**
Adding README file to the new ah_wireless
input plugin. Also correcting few calculations.

## Checklist

- [x] No AI generated code was used in this PR

## Related issues

resolves #

### CODE REVIEW

**Comment by arnelplim** [comment]
You need to bump extr_version.txt to 11.

File: plugins/inputs/ah_wireless/README.md [hunk 1]
```diff
@@ -0,0 +1,4381 @@
...
+On HiveOS, we can check output by executing hiden commands:
+_show report snapshot client
+and
+_show report snapshot interface
+
+## Example Output
+
+```shell
+{
+  'rfStats': [
```
Comment (arnelplim) [inline plugins/inputs/ah_wireless/README.md:332]: This sample output appears based on EXTR serializer format (i.e. data_format = "extr") as it leaves the output plugin, e.g. outputs.http.  I don't think output specific data format should be included in the README of an input plugin.  We should probably use standard influxdb format for your example, given the output format will vary depending on which output plugin this data is destined for.

See input plugin like disk or internal for their Example Output.

File: plugins/inputs/ah_wireless/ah_wireless.go [hunk 2]
```diff
@@ -82,18 +87,18 @@ func getHDDStat(fd int, ifname string) *ah_ieee80211_hdd_stats {
...
+		request.data.length = VAP_BUFF_SIZE
 
         copy(request.ifrn_name[:], ah_ifname_radio2vap(ifname))
 
-	offsetsMutex.Lock()
+		offsetsMutex.Lock()
 
         if err := ah_ioctl(uintptr(fd), IEEE80211_IOCTL_GENERIC_PARAM, uintptr(unsafe.Pointer(&request))); err != nil {
-                log.Printf("Open ioctl data error %s",err)
+                log.Printf("getHDDStat ioctl data error %s",err)
```
Comment (arnelplim) [inline plugins/inputs/ah_wireless/ah_wireless.go:97]: Consider using log.Errorf(), log.Warnf(), or log.Debugf() vs log.Printf().

**Review by arnelplim — APPROVED** [review]
Minor comments.  Looks good otherwise.

File: plugins/inputs/ah_wireless/ah_wireless.go (hunk 2 above)
Comment (chayan-04) [reply plugins/inputs/ah_wireless/ah_wireless.go:97]: I will cleanup and re-organize logging in next commit

File: plugins/inputs/ah_wireless/README.md (hunk 1 above)
Comment (chayan-04) [reply plugins/inputs/ah_wireless/README.md:332]: I will remove it.

**Pull request merged**